### PR TITLE
Gravity example with planets

### PR DESCRIPTION
**Pull request by PhilipRosedale**
This script creates two spheres with sizes proportional to the moon and earth, in front of you.  The moon object has a script *gravity.js" that applies a gravitational force.  If you grab the moon and gently throw it, you can get it to go into orbit around the earth.  As with real orbits, you will see that this takes some practice!  It is easy to throw it too far away (into a hyperbolic orbit, like a comet).   Once you have a feel for it, you will be able to get it to travel in a roughly circular orbit.

### CODE REVIEW

**Comment by PhilipRosedale** [comment]
Test plan:  Run makePlanets.js, see two spheres appear in front of you.  Near or far grab the smaller sphere and on release it will 'fall' toward the larger sphere.  By grabbing and tossing it, you should be able to get it to orbit.

**Comment by hifi-gustavo** [comment]
Build console output for commit: c4d125fdb1fefe3de74d9bf95342c0902b8dc71e available at: https://gist.github.com/f0c075a26132badc55cc33afef0ff611<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Alpha-PR7694.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Alpha-PR7694.dmg

**Comment by Nex-Pro** [comment]
Behaved as per description. Was able to observe orbital motion initiated by @thoys . 
Moon was affected by whichever sphere was nearest.
Also displayed behaviour where the initial path was influenced by one earth then went into orbit around second earth.
Works well.

File: examples/NBody/gravity.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        return (count == 0) ? 0 : totalTime / count;
+    }
+    this.elapsed = function() {
+        return new Date().getTime() - time;
+    }
+}
+
+(function () {
+	var entityID,
+    wantDebug = true,
```
Comment (ZappoMan) [inline examples/NBody/gravity.js]: indent is messed up here

File: examples/NBody/makePlanets.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+var earth = null;
+var moon = null;
+
+var SCRIPT_URL = Script.resolvePath("gravity.js");
+
+HIFI_PUBLIC_BUCKET = "http://s3.amazonaws.com/hifi-public/";
+Script.include(["../libraries/toolBars.js"]);
+var toolBar = new ToolBar(0, 0, ToolBar.HORIZONTAL, "highfidelity.makePlanets.js", function(screenSize) {
+  return {
```
Comment (ZappoMan) [inline examples/NBody/makePlanets.js]: indent is messed up here - coding standard says 4 spaces

File: examples/NBody/makePlanets.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+//   
+
+var SCALE = 3.0;
+var EARTH_SIZE = 3.959 / SCALE;
+var MOON_SIZE = 1.079 / SCALE;
+
+var BUTTON_SIZE = 32;
+var PADDING = 3;
+
+var screenSize = Controller.getViewportDimensions();
```
Comment (ZappoMan) [inline examples/NBody/makePlanets.js]: you don't really want to do this, as it won't work well in the HMD... you want to use the soon to be available `Controller.getRecommendedOverlayRect()` to get the rectangle for where your overlays will look best and position from there.

**Comment by sethalves** [comment]
I'll put up a new version of this, soon.

**Pull request closed without merging**